src/components/LineChart/LineChart.jsx

```diff
@@ -474,6 +487,29 @@ class LineChart extends PureComponent {
     return `${yAxisLabel}${yAxisUnit ? ` (${yAxisUnit})` : ''}`;
   }
 
+  /**
+   * Add incident annotations to the graph
+   */
+  addAnnotation(description, x, y, dx, dy) {
+    const annotation = [
```

Comment:
I think this could be named more descriptively, which might make line 507-508 more readable.

Reply:
Added more info to the docstring, does that help?